README.md

```diff
@@ -1,5 +1,13 @@
 # eslint-plugin-externalincludes
 
+![Lifecycle Active](https://badgen.net/badge/Lifecycle/Active/green)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![npm version](https://badge.fury.io/js/%40morgan-stanley%2Feslint-plugin-externalincludes.svg)](https://www.npmjs.com/package/@morgan-stanley/eslint-plugin-externalincludes)
+[![Build Status](https://github.com/MorganStanley/desktopJS/actions/workflows/continuous-integration.yml/badge.svg?event=push)](https://github.com/MorganStanley/desktopJS/actions/workflows/continuous-integration.yml)
```

Comment:
Wrong build status badge

Reply:
Fixed.  Coped markdown from badge generator of the action in GitHub.